backend/src/test/java/com/github/esgoet/backend/controllers/BookControllerIntegrationTest.java

```diff
@@ -111,4 +123,43 @@ public void deleteBook() throws Exception {
                 .andExpect(content().json("[]"));
     }
 
+    @DirtiesContext
+    @Test
+    void updateBook_Test_When_IdMatches() throws Exception {
+        // GIVEN
+        bookRepository.save(new Book("1", "author1", "title1", Genre.FANTASY, "description1", "12345678", "cover1", localDate));
+
+        Book updatedBook = new Book("1", "author2", "title2", Genre.HISTORY, "description2", "23456789", "cover2", localDate);
+
+        // WHEN
+        mockMvc.perform(MockMvcRequestBuilders.put("/api/books/1/update")
+                        .contentType("application/json")
+                        .content("""
+                                {
+                                    "id": "1",
```

Comment:
Do we need to update (or change) the value of `id`? If the `id` is not changed after creation, do we need to add it as a payload for the PUT method?